packages/url_launcher/lib/url_launcher.dart

```diff
@@ -30,7 +30,7 @@ const MethodChannel _channel =
 ///
 /// Note that if any of the above are set to true but the URL is not a web URL,
 /// this will throw a [PlatformException].
-Future<Null> launch(
+Future<void> launch(
```

Comment:
We could use async/await here so we don't need to change the return type, right?

Reply:
Yep. I considered it, but then I looked at flutter gallery sources and saw that nobody depends on the type anyway